src/dodal/stubs/flyables.py

```diff
@@ -0,0 +1,60 @@
+import bluesky.plan_stubs as bps
+from bluesky.protocols import Flyable
+
+from dodal.common import MsgGenerator, group_uuid
+
+
+def fly_and_collect(
+    flyer: Flyable,
```

Comment:
Should: The docstring says this should implement `Flyable` and `Collectable`, can the type hint reflect that?

Reply:
Adjusted, FlyableCollectable added to core.types, it's a useful construction for how we're using hardware scans.